src/libraries/Microsoft.PowerFx.Core/Lexer/Tokens/CommentToken.cs

```diff
@@ -8,17 +8,16 @@ namespace Microsoft.PowerFx.Core.Lexer.Tokens
 {
     internal class CommentToken : Token
     {
-        private readonly string _val;
         public bool IsOpenBlock;
 
         public CommentToken(string val, Span span)
             : base(TokKind.Comment, span)
         {
             Contracts.AssertValue(val);
-            _val = val;
```

Comment:
>_val [](http://example.com/codeflow?start=12&length=4)

Why this change?
Is there something wrong with a private field?

Reply:
auto applying https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0032

Nothing wrong with a private field, but if we're adding style rules, we might as well have consistent application of them.

Reply:
Can you clarify what "consistent application" means.  Eg, do the new style rules discourage private fields?

Reply:
That style rule discourages using a private field that's just a backing for a public property, yes.